src/app/header/header.component.ts

```diff
@@ -24,6 +24,8 @@ export class HeaderComponent implements OnInit {
         if (!this.cart.cartItems) {
           this.cart.cartPrice = 0;
           this.cart.cartPrice = this.cart.cartPrice.toFixed(2);
+        } else {
+          this.cart.cartPrice = this.cart.cartPrice.toFixed(2);
```

Comment:
why don't you just move the line out of the if since it's happening no matter what